client/components/support-info/style.scss

```diff
@@ -0,0 +1,13 @@
+.support-info {
+	float: right;
+	margin: 0 0 20px 20px;
+}
+
+.popover {
+	.support-info__privacy {
+		display: block;
+		margin-top: 14px;
+		padding-top: 12px;
+		border-top: 1px solid rgba( $gray, 0.5 );
```

Comment:
We should rather use a color variable here:

https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/_colors.scss#L10

Anecdotally, in the past I've used `$gray-lighten-30` as a divider border.